index.js

```diff
@@ -78,16 +78,16 @@ const receiveLogin = createActionCreator(
 
 const logout = createActionCreator(
     'LOGOUT',
-    user => ({ user })
+    () => {}
 )
 
 const authReducer = combineActionHandlers(null, {
-  [receiveLogin]: (state, action) => action.users.find(u => u.login === action.user && u.password === action.password) !== undefined ? action.user : null,
-  [logout]: (state, action) => null
+  [receiveLogin]: (state, {user, password, users}) => users.find(u => u.login === user && u.password === password) !== undefined ? user : null,
```

Comment:
What about [this comment](https://github.com/BCedric/redux-book-shop/pull/18#discussion_r101085942)??